PRIVACY_POLICY.md

```
@@ -28,9 +28,9 @@ NotebookLM Citation Mapper is a Chrome extension that enhances your experience w
 - **What we access:** Permission to write text to your system clipboard
 - **What we don't do:** We never read from your clipboard or access clipboard history
 
-#### `host_permissions` for `https://notebooklm.google.com/*`
+#### `host_permissions` for `https://notebook.google.com/*`
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)